metadata_backend/api/translator.py

```diff
@@ -1,8 +1,8 @@
 """Handle translating API endpoint functionalities to CRUD operations."""
-
 from typing import Dict
 
 from aiohttp import web
+from bson import json_util
```

Comment:
more of a question: i see this is used just for `dumps` is there a difference between `json.dumps` ?

Reply:
Yeah, json.dumps crashes when trying to serialize data returned from Mongo. More info here: https://api.mongodb.com/python/current/api/bson/json_util.html

Reply:
ok. was just wondering.